instructions.md

```diff
@@ -10,14 +10,16 @@ suite.
 ## Running
 
 The standard tool for running these tests is
-[sonobuoy](https://github.com/heptio/sonobuoy), and the standard way to run
-these in your cluster is with `curl -L https://raw.githubusercontent.com/cncf/k8s-conformance/master/sonobuoy-conformance-1.7.yaml | kubectl apply -f -`.
+[sonobuoy](https://github.com/heptio/sonobuoy/tree/latest), and the standard way to run
+these in your cluster is with `curl -L https://raw.githubusercontent.com/heptio/sonobuoy/latest/examples/conformance.yaml | kubectl apply -f -`.
```

Comment:
Would prefer we kept this yaml file in this repo.

Reply:
I'd prefer to point to the latest/example which will be kept up to date.  The reason for the change was confusion that arose from #17 and #6 .

Reply:
I don't understand why it's easier to keep the one in the sonobuoy repo up to date than this one.  I agree that the config should be kept up to date, but I don't see why that means it can't be in this repo.

I think that our instructions should be under a cncf repo, and the sonobuoy config is part of those instructions.  I would prefer that revisions to the YAML config undergo review here.

For example, the linked file includes a long list of resources and the systemd_logs plugin, which users don't need to run in order to generate conformance results.

Reply:
> I don't understand why it's easier to keep the one in the sonobuoy repo up to date than this one. 

b/c it's the canonical source and actively maintained and kept up to date, I'd also be happy to trim the conformance one and call it cncf-conformance.  If we keep it here, that means someone should maintain it and actively update and verify it.  The reason for this PR is because there were two issues filed against the version in the repo and no answers were given.